modules/ocf_desktop/files/xsession/print-notify-handler

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+import functools
+import os
+from configparser import ConfigParser
+
+import redis
+
+
+redis_connection = functools.partial(redis.StrictRedis,
+                                     host='broker',
+                                     port=6378,
+                                     ssl=True,
+                                     db=0,
+                                     password=None)
```

Comment:
maybe format this so the arguments start at the next line and one indent back? I think that's a style we use elsewhere